src/reporters/ClearTextReporter.ts

```diff
@@ -55,6 +55,38 @@ export default class ClearTextReporter implements Reporter {
     this.writeLine('Mutation score based on all code: ' + codebaseColor(mutationScoreCodebase + '%'));
   }
 
+  private logMutantResult(result: MutantResult): void {
+    this.writeLine(result.sourceFilePath + ': line ' + result.location.start.line + ':' + result.location.start.column);
+    this.writeLine('Mutator: ' + result.mutatorName);
+    this.writeLine(chalk.red('-   ' + result.originalLines));
+    this.writeLine(chalk.green('+   ' + result.mutatedLines));
+    this.writeLine('\n');
+    this.writeLine('Tests ran: ');
+    _.forEach(result.testsRan, (spec: string) => {
+      this.writeLine('    ' + spec);
+    });
+    this.writeLine('\n');
+  }
+
+  private logMutantResultToDebug(result: MutantResult): void {
```

Comment:
This method looks a lot like logMutantResult. Can't we combine the 2? As a second argument we could give the log function for example